public/js/main.js

```diff
@@ -0,0 +1,128 @@
+const speakersContainer = document.getElementById('speakers-card-container');
+const speakersBtn = document.getElementById('speakers-btn');
+let speakerShowMore = false;
+
+const speakersList = [
+  {
+    id: 0,
+    imgUrl: './public/images/speaker-1.jpg',
+    author: 'Kaium Uddin',
+    position: 'Cras volutpat, massa id vel, diam magna',
+    desc: 'Maecenas vel lorem interdum, fermentum quam vel, posuere ante.',
+  },
+  {
+    id: 1,
+    imgUrl: './public/images/speaker-2.jpg',
+    author: 'Mohima Rahman',
+    position: 'Cras volutpat, massa id vel, diam magna',
+    desc: 'Maecenas vel lorem interdum, fermentum quam vel, posuere ante.',
+  },
+  {
+    id: 2,
+    imgUrl: './public/images/speaker-3.jpg',
+    author: 'Monir Hossain',
+    position: 'Cras volutpat, massa id vel, diam magna',
+    desc: 'Maecenas vel lorem interdum, fermentum quam vel, posuere ante.',
+  },
+  {
+    id: 3,
+    imgUrl: './public/images/speaker-4.jpg',
+    author: 'Sharif Khan',
+    position: 'Cras volutpat, massa id vel, diam magna',
+    desc: 'Maecenas vel lorem interdum, fermentum quam vel, posuere ante.',
+  },
+  {
+    id: 4,
+    imgUrl: './public/images/speaker-5.jpg',
+    author: 'Nusrath Jahan',
+    position: 'Cras volutpat, massa id vel, diam magna',
+    desc: 'Maecenas vel lorem interdum, fermentum quam vel, posuere ante.',
+  },
+  {
+    id: 5,
+    imgUrl: './public/images/speaker-6.jpg',
+    author: 'Esrath Jahan',
+    position: 'Cras volutpat, massa id vel, diam magna',
+    desc: 'Maecenas vel lorem interdum, fermentum quam vel, posuere ante.',
+  },
+];
+
+const toggleMenu = () => {
+  mobileMenu.classList.toggle('hidden');
+  mobileMenu.classList.toggle('flex');
+};
+
+const generateSpeakers = (n = 2) => {
+  let cardsHtmlGenerator = '';
+  for (let i = 0; i < n; i += 1) {
+    cardsHtmlGenerator += `
+  <!-- speakers card ${speakersList[i].id} -->
+  <div class="speakers-card relative mt-10 flex items-center gap-4">
+    <div class="speakers-card-img-wrapper relative flex max-w-fit items-center justify-end">
+      <img
+        src="${speakersList[i].imgUrl}"
+        alt="speaker-${speakersList[i].id}: ${speakersList[i].author}"
+        class="speakers-card-img relative w-[75%]"
+      />
+      <img
+        src="./public/images/speaker-bg.png"
+        alt="speaker background"
+        class="absolute -top-[20%] right-[50%] -z-10 w-[40%]"
+      />
+    </div>
+    <div class="speakers-card-content flex max-w-[60%] flex-col gap-3">
+      <h3 class="speakers-card-title text-2xl font-bold text-heavy-metal">
+        ${speakersList[i].author}
+      </h3>
+      <p class="speakers-card-subtitle l font-medium italic leading-6 tracking-wide text-coral-red">
+        ${speakersList[i].position}
+      </p>
+      <hr class="h-[2px] w-7 bg-metal-grey bg-opacity-60" />
+      <p class="speakers-card-text text-metal-grey">
+      ${speakersList[i].desc}
+      </p>
+    </div>
+  </div>
```

Comment:
- Hey, great job with the styling here, however, it could be improved to improve the user experience on the mobile version when the user tends to scroll the section of your website. Your design is great, however, we can try reduce the gaps, font size, and weight, and also try to reduce the text content in the description so that our design can be close to the design template. Currently, yours is having too much space, let's try improving the styles to be close to the design. see the comparison below :+1: 
- Also, update the color of the small line to match the color used in the design


*design template*
![Screenshot from 2022-11-25 17-03-06](https://user-images.githubusercontent.com/59397159/204022749-9512f5d9-2e57-4b70-abf8-c9a211148d61.png)

*your design*
![Screenshot from 2022-11-25 17-01-58](https://user-images.githubusercontent.com/59397159/204022755-3ae50876-f9e7-408b-a270-53dd2b7d4c94.png)

- As you can see from the template, pay attention to the spaces, and height of each card
- You can now just one card of yours is big. Please, fix this :pray: